packages/@okta/vuepress-site/docs/guides/express-configuration/main/index.md

```diff
@@ -28,7 +28,7 @@ After confirming your app's eligibility, the Okta Express Configuration team pro
 
 Express Configuration allows an Okta admin to automate the setup of an OIN OIDC instance integration. This leads to faster, secure, and scalable SaaS deployments:
 
-* **Faster configuration**: Reduces time required to set up an instance of OIN OIDC integration by automating the exchange of configuration information between Okta and Auth0. It supports OIDC capabilities through a single control panel, significantly speeding up the overall configuration process for OIN OIDC integration instances.
-* **Secure data handling**: Uses OAuth 2.0 consent flows for secure and authorized sharing of sensitive configuration data. This approach also reduces potential errors related to credentials and configuration settings.
-* **Scalable solution**: Simplifies and standardizes the integration deployment process, which makes it more scalable. The automated workflow allows for consistent and repeatable deployments of OIN OIDC integrations across multiple customers or environments, supporting a scalable app ecosystem.
-* **Improved configuration experience**: Provides an intuitive and efficient configuration experience. The automation removes the complexity of manual setup, which allows Okta customer admins to quickly add instances of Auth0-enabled OIN integrations.
\ No newline at end of file
+* **Faster configuration**: Express Configuration reduces time required to set up an instance of OIN OIDC integration by automating the exchange of configuration information between Okta and Auth0. It supports OIDC capabilities through a single control panel, significantly speeding up the overall configuration process for OIN OIDC integration instances.
```

Comment:
Adding Express Configuration to all of these is repetitive, since it's mentioned in the stem sentence above. Was this an Acrolinx fix? It's better the way it was.